core/trino-main/src/main/java/io/trino/execution/scheduler/NodeSchedulerConfig.java

```diff
@@ -108,8 +109,20 @@ public NodeSchedulerConfig setIncludeCoordinator(boolean includeCoordinator)
         return this;
     }
 
-    @Config("node-scheduler.max-pending-splits-per-task")
-    @LegacyConfig({"node-scheduler.max-pending-splits-per-node-per-task", "node-scheduler.max-pending-splits-per-node-per-stage"})
+    @Config("node-scheduler.min-pending-splits-per-task")
+    @LegacyConfig({"node-scheduler.max-pending-splits-per-task", "node-scheduler.max-pending-splits-per-node-per-task", "node-scheduler.max-pending-splits-per-node-per-stage"})
+    public NodeSchedulerConfig setMinPendingSplitsPerTask(int minPendingSplitsPerTask)
+    {
+        this.minPendingSplitsPerTask = minPendingSplitsPerTask;
+        return this;
+    }
+
+    public int getMinPendingSplitsPerTask()
+    {
+        return minPendingSplitsPerTask;
+    }
+
+    @Config("node-scheduler.max-adjusted-pending-splits-per-task")
     public NodeSchedulerConfig setMaxPendingSplitsPerTask(int maxPendingSplitsPerTask)
```

Comment:
`setMaxPendingSplitsPerTask` -> `setMaxAdjustedPendingSplitsPerTask`
and same for getter